Source/RomanNumeral.h

```diff
@@ -12,6 +12,29 @@
  * @param input The number to convert.
  * @param output String buffer in which to store the converted Roman numeral.
  */
+
+typedef struct {
+    int num;
+    const char * romanNumeral;
+} TableEntry_t;
```

Comment:
You'll want to keep this guy in the C file as well since it's not necessary for any outside entities to know about this type.